e2e/app/page/notebook-page.ts

```diff
@@ -235,6 +237,28 @@ export default class NotebookPage extends AuthenticatedPage {
     await analysisPage.deleteResource(notebookName, ResourceCard.Notebook);
   }
 
+  /**
+   * Delete runtime
+   */
+  async deleteRuntime(): Promise<void> {
```

Comment:
Why is this duplicated here?

Reply:
This runtime sidebar is accessible from the Jupyter Notebook page. Behavior is different when the NotebookPreviewPage is opened automatically. Line #252 and #253 are making sure page synchronized.